src/test/component.test.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from '../components/StockList';
+import Nav from '../components/Nav';
+import store from '../redux/configureStore';
+
```

Comment:
- [ ] OPTIONAL
As this activity requires, kindly ensure to include unit tests for this project